### DISCUSSION TITLE
在dock栏增加鼠标悬停时聚焦提示，类似windows的鼠标滑过dock栏的浅色背景

### DISCUSSION THREAD

**Discussion by xinguankeli** [linuxdeepin/developer-center#4671] [Features Request & Ideas | 特性请求 & 头脑风暴]
目前觉着dde-dock的dock栏鼠标滑过时候没有聚焦，我眼睛看不到大概的位置，需要类似windows10的那种小动作

**Comment by BLumia** [comment]
关联： https://github.com/linuxdeepin/dde-dock/pull/874
cc @meiyixiang
